src/web/components/Card/components/CardListItem.tsx

```diff
@@ -0,0 +1,61 @@
+import React from 'react';
+import { css, cx } from 'emotion';
+
+import { palette } from '@guardian/src-foundations';
+
+const listStyles = (percentage?: CardPercentageType) => css`
+    display: flex;
+    flex-grow: 1;
+
+    /* Here we ensure the card stretches to the given percentage */
+    flex-basis: ${percentage && percentage};
+
+    /* We absolutely position the vertical divider and this contains that */
+    position: relative;
+
+    /* Set spacing on the li element */
+    padding-left: 10px;
+    padding-right: 10px;
+`;
+
+const marginBottomStyles = css`
+    margin-bottom: 10px;
+`;
+
+const verticalDivider = css`
+    :before {
+        content: '';
+        display: block;
+        position: absolute;
+        top: 0;
+        bottom: 0;
+        left: 0;
+        width: 0.0625rem;
```

Comment:
Should this width not just be a px rather than a rem? I assume we don't want it to get fatter?

Reply:
Also, you seem to have a component for this?

Reply:
`CardListItem` shouldn't be there 🤦‍♂ . I've added a commit to remove it.